test/double_check_test.dart

```diff
@@ -31,6 +33,16 @@ void main() {
       // newline normalization issues.
       testOn: "!windows");
 
+  test("deprecations are up-to-date", () {
+    var inputText = File(deprecations.yamlPath).readAsStringSync();
+    var outputText = File(deprecations.dartPath).readAsStringSync();
+    var checksum = sha1.convert(utf8.encode(inputText));
+    if (!outputText.contains('// Checksum: $checksum')) {
+      fail('${deprecations.dartPath} is out-of-date.\n'
+          'Run `dart run grinder` to update it.');
+    }
+  }, testOn: "!windows");
```

Comment:
Probably worth including the above comment here as well, or moving this condition onto a group that contains both up-to-date tests.

Reply:
Done